### DISCUSSION TITLE
Recommended received noise value for ray tracing scenario

### DISCUSSION THREAD

**Discussion by charaf412** [NVlabs/sionna#315] [General]
Hello, 

I used this [ray tracing tutorial](https://nvlabs.github.io/sionna/tutorials.html#ray-tracing) to calculate the channel gains. Now, I would like to calculate my SNR. For that, I need to know the variance of the ambient noise (received noise). 

What is the recommended value for this type of scenario?

Thank you.

**Comment by jhoydis** [comment]
The thermal noise power is typically computed as $W N_o$, where $W$ is the bandwidth in Hz and $N_o$ is the noise power spectral density. $N_o$ depends on the temperature: $N_o=k T$, where $k$ is the Boltzmann constant and $T$ the temperature in Kelvins (K). A typical value would be $T=293$ K (which is 20 degrees Celsius).

You would also need to assume a transmit power P. The SNR would then be computed as
$SNR = \frac{g P }{W N_o}$
where $g$ is the channel gain.
